package/base-files/files/lib/upgrade/common.sh

```diff
@@ -217,11 +217,16 @@ indicate_upgrade() {
 # $(1): path to image
 # $(2): (optional) pipe command to extract firmware, e.g. dd bs=n skip=m
 default_do_upgrade() {
+	local err
```

Comment:
Changing to this file should be put into separated commit.

Reply:
_(Change was moved to appropriate commit.)_